### ISSUE TITLE
[Bug] Index length limit

### ISSUE THREAD

**Issue by azlux** [urlhum/UrlHum#48]
**Describe the bug**
The installation fail during the database creation.

```
www-data@urlhum:~/html$ php artisan migrate
**************************************
*     Application In Production!     *
**************************************

 Do you really wish to run this command? (yes/no) [no]:
 > yes

Migration table created successfully.
Migrating: 2014_10_12_000000_create_users_table

   Illuminate\Database\QueryException  : SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes. (SQL: alter table `users` add unique `users_email_unique`(`email`))
```

**To Reproduce**
1. Use mariadb database
this limitation is link to the default database innoDB and the char-set you use (utf8mb4_bin)

The best is to reduce the index length or to use the char-set utf8_bin.

Best regards,
Az

**Comment by Azocki** [comment]
do you have discord?
i know this issue simple to fix the error say it already.

**Comment by optiroot** [comment]
Thank you for creating this issue.
This issue is pretty common with Laravel and MariaDB: Will work on a fix as soon as possible.
Thanks!

**Comment by azlux** [comment]
@Azocki Yes I have it. You can send me invation with my github email (on my public profile)

**Comment by Azocki** [comment]
hey optiroot, you dont need to fix this, you just increase the limit

**Comment by Azocki** [comment]
@azlux add me on discord: ! Hiro い 💉#5943

**Comment by azlux** [comment]
@Azocki I think you need a fix (or at least a change in the wiki ) because it's not on the doc.

On my case, my database is used my multiple services. I hope increasing some limit will not break anything.

**Comment by Azocki** [comment]
noo

**Comment by Azocki** [comment]
its not increasing db, its fixing in the laravel boot file

**Comment by azlux** [comment]
@Azocki I'm not able to add you on discord, mine is Azlux#9825

**Comment by Azocki** [comment]
added

**Comment by optiroot** [comment]
For that purpose I need fix the Laravel boot file.
Please, don't use this issue as a discussion forum. If you want to help each other using a forum-like community you can use [UrlHum Docs center](https://urlhum.readme.io/discuss-new).

Thanks!

**Comment by Azocki** [comment]
yeah, optiroot i told him that im gonna fix it for him.

**Comment by optiroot** [comment]
Fixed in #51.
